pjsip/include/pjsua-lib/pjsua.h

```diff
@@ -4329,6 +4329,15 @@ typedef struct pjsua_acc_config
      */
     pjmedia_rtcp_fb_setting rtcp_fb_cfg;
 
+#if defined(PJMEDIA_HAS_RTCP_XR) && (PJMEDIA_HAS_RTCP_XR != 0)
```

Comment:
why in pjsua is this protected by #if macro but not in pjsua2?

looking at the other settings (such as video, srtp), it seems that typically settings do not need macro protection

Reply:
Ah right, added that because there is macro protection of `PJMEDIA_STREAM_ENABLE_KA` few lines up.
Will update the patch.